cirq-superstaq/pyproject.toml

```diff
@@ -0,0 +1,35 @@
+[project]
+name = "cirq-superstaq"
+description = "The Cirq module that provides tools and access to Superstaq."
+authors = [
+  { name = "Superstaq development team", email = "superstaq@infleqtion.com" },
+]
+readme = { file = "README.md", content-type = "text/markdown" }
+license = { text = "Apache 2" }
+requires-python = ">=3.8.0"
+dynamic = ["version", "dependencies", "optional-dependencies"]
+
+[project.urls]
+homepage = "https://github.com/Infleqtion/client-superstaq"
+
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools.packages.find]
+include = ["cirq_superstaq*"]
+
+[tool.setuptools.package-data]
+cirq_superstaq = ["py.typed"]
+
+[tool.setuptools.dynamic.version]
+attr = "cirq_superstaq._version.__version__"
+
+[tool.setuptools.dynamic.dependencies]
+file = ["requirements.txt"]
+
+[tool.setuptools.dynamic.optional-dependencies.dev]
+file = ["dev-requirements.txt"]
```

Comment:
looks like it works

Reply:
nice! we shouldn't remove `optional-dependencies.examples` though - basically we just want `pip install x-superstaq[dev]` to be a strict superset of `x-superstaq[examples]`, instead of having to use `x-superstaq[dev, examples]` or duplicating requirements in example-requirements.txt and dev-requirements.txt

Reply:
relatedly, we should remove qiskit-terra and symengine from cirq-superstaq/dev-requirements.txt

Reply:
cool, done

Reply:
sorry one more thing: the one instance of "[dev,examples]" can now be changed to just "[dev]" in ci.yml